pkg/operator/podsecurityreadinesscontroller/conditions.go

```diff
@@ -0,0 +1,89 @@
+package podsecurityreadinesscontroller
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	operatorv1 "github.com/openshift/api/operator/v1"
+	"github.com/openshift/library-go/pkg/operator/v1helpers"
+)
+
+const (
+	ViolationReason = "PodSecurityReadinessController"
```

Comment:
this does not look like a ViolationReason

Reply:
ViolationReasonRepoter is being better?